torchci/clickhouse_queries/compilers_benchmark_performance/query.sql

```diff
@@ -144,6 +142,62 @@ results AS (
         LEFT JOIN accuracy_results ON performance_results.name = accuracy_results.name
         AND performance_results.replaced_filename = accuracy_results.replaced_filename
         AND performance_results.workflow_id = accuracy_results.workflow_id
+    WHERE
+        accuracy != 'model_fail_to_load'
+        AND accuracy != 'eager_fail_to_run'
+),
+-- This is to accommodate cases where only accuracy results are available, i.e. export
```

Comment:
I discover another regression here where export results were missing from the dashboard.  That's a special case where there is only accuracy result for `torch.export`.